src/views/PurchaseOrder.vue

```diff
@@ -153,6 +153,13 @@ export default defineComponent({
         this.file = event.target.files[0];
         if(this.file){
           showToast(translate("File uploaded successfully"));
+          this.fieldMapping = {
+            orderId: "",
+            productSku: "",
+            orderDate: "",
+            quantity: "",
+            facility: "",
+          }
```

Comment:
We can also iterate over the fieldMapping object and delete all the properties instead of hardcoding, like this- `Object.keys(this.fieldMapping).map(key => { delete this.fieldMapping[key] })`, what do you think @shashwatbangar?